src/printer.js

```diff
@@ -628,15 +628,17 @@ function genericPrintNoParens(path, options, print) {
           concat([
             leftBrace,
             indent(
-              options.tabWidth * (parentIsUnionTypeAnnotation ? 2 : 1),
+              options.tabWidth + (parentIsUnionTypeAnnotation ? 3 : 0),
```

Comment:
@rhengles I agree with you. This is how I tried to make it work in the first place with a different calculation. What's your opinion on this @jlongster?